resume-buddy/src/main/webapp/resume-review-script.js

```diff
@@ -30,17 +47,21 @@ function createListElement(date, type, text, id) {
 
   containerDiv.className = 'comment-container';
   const typeText = document.createElement('b');
-  typeText.innerText = type + " ";
+  typeText.innerText = type + ":  ";
   liElement.appendChild(typeText);
 
-  const dateNode = document.createElement('i');
-  dateNode.innerText = date + " ";
-  liElement.appendChild(dateNode);
-
-  const textNode = document.createTextNode(text);
+  const textNode = document.createTextNode(text + " ");
   liElement.appendChild(textNode);
 
-  // TODO: Add delete button next to comment which deletes comment
+  const deleteButton = document.createElement('button');
+  deleteButton.innerHTML = '&#10005;';
+  deleteButton.className = "delete-button";
+  deleteButton.style.cssFloat = "right";
```

Comment:
It looks like `delete-button` isn't in the CSS unless I missed it. What do you think about adding it and then moving the `float: right` to there?